pkg/handlers/nfs_handler.go

```diff
@@ -127,6 +127,10 @@ func (h *nfsHandler) OnDelete(ns string, vc kvcv1.VolumeConfig, controllerRef me
 
 		for _, resource := range resourceList {
 			resControllerRef := metav1.GetControllerOf(resource)
+			if resControllerRef == nil {
```

Comment:
Does this mean we should just delete it?

Reply:
Or maybe we should handle this in the reconciler. Can you add a TODO there.

Reply:
No we should not delete these resources. They should not manipulated. This case is true when a resource has the same labels as a volume manager CR.